portal/_static/custom.css

```
@@ -1,3 +1,22 @@
+main.banner-main #project-pythia {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+main.banner-main #project-pythia p {
+  font-size: 1.4rem;  /* default: 1.25rem */
+  /* font-weight: 700; default: 300 */
+}
+
+main.banner-main #project-pythia a,
+main.banner-main #project-pythia a:visited {
+  text-decoration: underline dotted rgba(var(--spt-color-gray-400), 1);
```

Whoops!  I screwed that up.  😄 

```suggestion
  color: rgba(var(--spt-color-light), 1);
  text-decoration: underline dotted rgba(var(--spt-color-gray-400), 1);
```